stereo_image_proc/CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package stereo_image_proc
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+5.0.1 (####-##-##)
+------------------
+* Breaking change: deleted full_dp parameter and created new parameter sgbm_mode to select the algorithm for stereo matching (`https://github.com/ros-perception/image_pipeline/issues/582`)
+* Contributors: Pablo David Aranda
```

Comment:
ah - this part is autogenerated - we can revert the changes to this file (it will get added when I do the next release - but editing it manually like this actually breaks the release tools).

The file I was asking to update is the actual documentation that we use for big changes between releases (so that when you upgrade from say humble/iron->jazzy, you know what changed) - that gets rendered here: http://docs.ros.org/en/rolling/p/image_pipeline/changelog.html - and comes from the file previously mentioned here https://github.com/ros-perception/image_pipeline/blob/rolling/image_pipeline/doc/changelog.rst - and yes, it's in a different "package" - because image_pipeline is the "metapackage" that rolls up all of these. When the Jazzy release gets closer, we'll end up also linking this page into the main documentation - in a page like this: http://docs.ros.org/en/rolling/Releases/Release-Iron-Irwini.html#changes-since-the-humble-release

I've added notes about this to https://github.com/ros-perception/image_pipeline/issues/946 - where I'm going to update our CONTRIBUTIONS.md file so these sort of things are better documented

Reply:
Okay, in the meanwhile I will add it to that file in the section that describes the changes between Iron and Jazzy, but the changes are being merged into the rolling branch, should I change the target?

Reply:
Rolling is the correct branch - Jazzy doesn't exist yet (the buildfarm will come on line in probably 3-4 weeks, and then we can make initial releases into Jazzy, although we typically wait to fork off a branch until we are really forced to since every extra branch adds to the maintainer overhead)

Reply:
Perfect, all the changes are done then.